OpenEdX/Managers/DeepLinkManager/DeepLinkManager.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DeepLinkManager.swift
+//  OpenEdX
+//
+//  Created by Anton Yarmolenka on 24/01/2024.
+//
+
+import Foundation
+import Core
+import UIKit
+
+public protocol DeepLinkService {
+    func configureWith(launchOptions: [UIApplication.LaunchOptionsKey: Any]?)
+    func handledURLWith(app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any]) -> Bool
+}
+
+class DeepLinkManager {
+    private var services: [DeepLinkService] = []
+    
+    // Init manager
+    public init(config: ConfigProtocol) {
+        services = servicesFor(config: config)
```

Comment:
How about renaming `servicesFor` to something like `configureServices(_ with config)` and call it. The method will populate `services` for all enabled services?

Reply:
I don't agree with this name - this func doesn't configure services but just inits and returns services for current config

Reply:
ok